examples_tests/61.UI/UI_System.hpp

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include <nabla.h>
+
+#include <glm/vec3.hpp>
+
+#include "../common/CommonAPI.h"
```

Comment:
you cannot depend on `CommonAPI` in an extension

Reply:
I need access to the keyboard and mouse inputs. I'm currently using the input system.  What should I do instead?

Reply:
take a `const IMouseEventChannel::range_t& events` as an argument

So basically whoever is using your GUI system, will be in charge of "filtering" the events, and only giving you the ones you should be interested in.